run.sh

```diff
@@ -2,6 +2,47 @@
 
 set -o errexit
 
+usage() {
+    echo "deploy               -- Deploy airship"
+    echo "update_openstack     -- Update openstack deployment"
+    echo "add_compute          -- Add compute node"
+    echo "remove_compute       -- Remove compute node"
+    echo "clean_all            -- Delete all resources related with deployment"
+    echo "clean_openstack      -- Delete resources that are in openstack namespace"
```

Comment:
I think we agreed that the purpose of clean_openstack is not clear, and implementation is half baked as the openstack site is still in Airship, and we only remove the pods etc. The user only needs "update OpenStack" is our conclusion. If we still hold the above opinion, I prefer we take out clean_openstack option.

Reply:
So, what is the preferred verb for removing deployment?  remove_deployment?

Reply:
> So, what is the preferred verb for removing deployment? remove_deployment?

I like remove_deployment than clean_all . but it does clean everything including images ?